app/scripts/lib/metaRPCClientFactory.test.js

```diff
@@ -9,13 +10,14 @@ describe('metaRPCClientFactory', () => {
     const metaRPCClient = metaRPCClientFactory(streamTest);
     metaRPCClient.foo();
   });
-  it('should be able to make an rpc request/response with the method and params and node-style callback', () => {
+  it('should be able to make an rpc request/response with the method and params and node-style callback', (done) => {
```

Comment:
Despite opinions documented in jest-eslint plugin, there's still valid usecases for the done callback.
To make these tests compatible with async/await style of asynchronous tests we'd have to use `new Promise` to create some convoluted wrappers with questionable error handling. 

This is the minimal change that makes these tests fail when assertions aren't met. Before this change putting`expect(result).toStrictEqual('NOPE');` below would not fail the test.